src/Common/src/System/Net/Logging/EmailEventSource.cs

```diff
@@ -10,18 +10,18 @@
 
 namespace System.Net
 {
-    [EventSource(Name = "Microsoft-System-Net-Mail", LocalizationResources = "FxResources.System.Net.Mail.SR")]
-    internal sealed class WebEventSource : EventSource
+    [EventSource(Name = "Microsoft-System-Net-Mail", Guid = "32663b9f-4d8d-44f8-9f29-f4efc12e6879", LocalizationResources = "FxResources.System.Net.Mail.SR")]
```

Comment:
>However you can also use *Microsoft-System-Net-Mail in most tools

What about tools where you have to specify the GUID?

And we already have GUIDs for other event sources.  See:
Documentation/debugging/windows-instructions.md